<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub3dtoolgui</class>
 <widget class="QWidget" name="sub3dtoolgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sub3dtoolgui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Input File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEditFileIn"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenFileIn">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEditFileOut"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenFileOut">
        <property name="text">
         <string>F</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>3D Options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="button3DSBS">
        <property name="text">
         <string>SBS</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3DTB">
        <property name="text">
         <string>TB</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3DN3D">
        <property name="text">
         <string>N3D</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Screen Resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="currentFont">
           <font>
            <family>FreeSans</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>Advance...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConvert">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpenFileIn</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>getFileIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpenFileOut</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>getFileOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DN3D</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeN3D()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DSBS</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeSBS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DTB</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeTB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>initGui()</slot>
  <slot>checkTool()</slot>
  <slot>getFileOut()</slot>
  <slot>getFileIn()</slot>
  <slot>change3DModeSBS()</slot>
  <slot>change3DModeTB()</slot>
  <slot>change3DModeN3D()</slot>
 </slots>
</ui>
